### ISSUE TITLE
Got Question

### ISSUE THREAD

**Issue by Logan7432** [huytd/agar.io-clone#316]
sooo uhh... im trying to run a server for school and i got it all working so for some reason i got no viruses :( and cant merge. but anyway is there a way to add the minimap to the server so i can find my friends or naw cause i see other servers have it like agario.biz

**Comment by abalabahaha** [comment]
Duplicate of #25 and #304. As for the minimap, I opened #317. But don't expect anything you're requesting to be done in 5 seconds, because we are not your personal army. If you want one of those, go pay someone.

**Comment by Logan7432** [comment]
Ok, I wish I knew how to code I would love to help work on this cause I have a server for school cause the site we use was agario.biz and now that's blocked so I made a unblocked website for it running this clone but it's not really fun because there's no viruses and we can't group back together, and I have other servers that work great that have everything I need, BUT I don't know how to connect to it without going on agario website (which is blocked) this is the only working one that me and my friends can connect to by just typing in IP:Port and not having to go on the main website. If you can plz help me.

**Comment by abalabahaha** [comment]
Sorry, can't help with that!

**Comment by IgorAntun** [comment]
@abalabahaha :clap:
